platform/onebox/backup_and_restore/clusterResources/export.sh

```diff
@@ -34,8 +34,8 @@ function backup_namespace() {
   declare -a NAMESPACES=()
 
   echo "$(date) Fetching list of all UUID namespaces"
-  readonly NAMESPACES=($(kubectl get ns -A | awk '/[0-9a-fA-F]{8}\-[0-9a-fA-F]{4}\-[0-9a-fA-F]{4}\-[0-9a-fA-F]{4}\-[0-9a-fA-F]{12}/ {print $1}'))
-  local namespaces_list=$(IFS=, ; echo "${NAMESPACES[*]}")
+  readonly NAMESPACES=$(kubectl get ns -A | awk '/[0-9a-fA-F]{8}\-[0-9a-fA-F]{4}\-[0-9a-fA-F]{4}\-[0-9a-fA-F]{4}\-[0-9a-fA-F]{12}/ {print $1}')
+  local namespaces_list=$(echo "$NAMESPACES" | paste -s -d, /dev/stdin)

```

Comment:
`paste -s -d "," /dev/stdin`
Is this the equivalent?

Reply:
there was bug in earlier, this handles it better. Tested by me